core/store/tx_manager.go

```diff
@@ -127,6 +127,22 @@ func (txm *EthTxManager) Connect(bn *models.Head) error {
 		txm.currentHead = *bn
 	}
 	txm.connected.Set()
+
+	// Upon connecting/reconnecting, rebroadcast any transactions that are still unconfirmed
+	attempts, err := txm.orm.UnconfirmedTxAttempts()
+	if err != nil {
+		return err
+	}
+
+	for _, attempt := range attempts {
+		logger.Infof("Rebroadcasting tx %v", attempt.Hash.String())
+
+		_, err := txm.SendRawTx(attempt.SignedRawTx)
+		if err != nil {
+			logger.Warnf("Failed to rebroadcast tx %v: %v", attempt.Hash, err)
```

Comment:
@se3000 @j16r We have a situation here where transaction attempts that have been confirmed while we've been offline might return an error on rebroadcast. We've opted to simply warn on this situation because we don't want to deter the reconnect. Please let me know if you have any thoughts on this.